### DISCUSSION TITLE
Tablet button bindings don't work

### DISCUSSION THREAD

**Discussion by h3ckt0** [ppy/osu#16984] [Q&A] [answered]
Neither with the build in tablet driver nor without it I can make use of any of my tablet buttons.
This makes playing on Lazer very annoying to me as I use a keypad, meaning I don't have access to my keyboard half the time. 
Is there anything i can do against that? When trying to assign any buttons - ofc - Lazer doesn't recognize me tapping the tablet buttons.

**Comment by peppy** [comment]
- What tablet are you using?
- What happens when you try and bind a key/button in the settings? Does it stay in a "waiting for press" state?

**Reply by h3ckt0** [reply]
-Gaomon S620
-Yeah it stays in "waiting for press" state (of course only when using said buttons, with keyboard everything is normal). My buttons are bound through OTD which works fine outside of Lazer but inside Lazer it just gets overwritten by something.

**Reply by peppy** [reply]
And to confirm, disabling the built-in tablet support doesn't help?

https://user-images.githubusercontent.com/191335/155661597-5a558fe4-f13e-47e6-9dcb-d8dffe0f257f.mp4

**Reply by h3ckt0** [reply]
That is correct.

**Reply by h3ckt0** [reply]
I don't have anything new on this but I do have a question if you ever happen to read this:

Why did you decide to "rework" or "nerf" hp in Lazer? Are you planning to keep hp the way it is in Lazer now? 

After playing for a few hours I realized there is a huge difference between Stable and Lazer - allowing me to pass maps I could only dream of passing on Stable. This so far has been the biggest letdown towards me staying on Lazer. 
It just feels very weird to have HP 6 feel a lot like HP 3 / 4. Classic mod did not change that, at least not as far as I'm concerned because I did pass an almost 8* map with classic turned on, not getting through 1/4th the way on Stable.

**Reply by bdach** [reply]
please do not post about hp in a discussion thread about tablet bindings

also see https://github.com/ppy/osu/issues/7975

**Reply by ZaneWood235** [reply]
I have the same issue and it's the only thing stopping me from playing Lazer. If it can't be fixed, then having the option to move the pause button from the bottom right to the bottom left would be nice because I'm left handed.

**Reply by peppy** [reply]
Can you provide more details? What happens when you try and bind pause? Can you show a video?

**Reply by ZaneWood235** [reply]
When I bind pause and try to press it, nothing happens. Not in-game or in the menus. Alphanumeric keys seem to work in the menus but similarly do not work in gameplay. (I provided a video, but I'm not really sure that it'll help explain very much)

https://github.com/ppy/osu/assets/136027785/9330e2d5-871b-4f4c-b20d-584640b1be7c

**Reply by peppy** [reply, marked as answer]
Have you tried binding the tablet button in-game using the settings interface? You might find that it works well if you do that.

**Reply by ZaneWood235** [reply]
I tried that just now, and it works! Some menu hotkeys only have one binding so that would suck for those who play standard and another game mode, but that isn't a dealbreaker for me.
